ClientLibrary/Snippets/Microsoft.TeamServices.Samples.Client/Wiki/WikisSample.cs

```diff
@@ -0,0 +1,217 @@
+﻿using Microsoft.TeamFoundation.SourceControl.WebApi;
+using Microsoft.TeamFoundation.Wiki.WebApi;
+using Microsoft.VisualStudio.Services.WebApi;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.TeamServices.Samples.Client.Wiki
+{
+    [ClientSample(WikiConstants.AreaName, WikiConstants.WikisResourceName)]
+    public class WikiV2Sample : ClientSample
```

Comment:
Maybe just call this `WikisSample` (i.e. don't say "V2", unless you plan on documenting V1 samples)